docs/content/blog/cncf.md

```diff
@@ -2,7 +2,34 @@
 title: "Porter Has Joined the CNCF"
 description: "Porter has been accepted into the Cloud Native Computing Foundation (CNCF)"
 date: "2020-09-24"
-draft: true
 ---
 
-Porter has been accepted into the Cloud Native Computing Foundation (CNCF)...
+You may have noticed a flurry of activity recently on the Porter repositories
+and wondered what was afoot. It was all in preparation for the big reveal:
+Porter has been accepted into the Cloud Native Computing Foundation (CNCF)!<!--more-->
+
+When Jeremy Rickard and I created Porter less than two years ago, this is what
+we hoped for. We wanted to create a user-friendly tool for that would make Cloud
+Native Application Bundles (CNAB) accessible and deployments less stressful. It
+had to be user-first and community-first, without vendors getting in the way. So
+our sights were set on finding an independent home at the CNCF from day one. It
+means a lot to us to have built a community around Porter and tricked the CNCF
+into letting us in. 😉
+
+Porter is a CNCF [sandbox] project which is the first step for early stage
+projects where we can continue to grow and experiment. We have always encouraged
```

Comment:
projects to have a public venue where we can continue to grow and experiment with a goal of having the best developer experience working with bundles. We welcome contributions from everyone in the community and provide a contribution ladder to encourage and support contributors.